tests/snuba/api/endpoints/test_group_details.py

```diff
@@ -166,7 +166,7 @@ def test_assigned_to_unknown(self):
         )
         assert response.status_code == 200
         response = self.client.put(
-            url, {"assignedTo": "user@doesntexist.com", "status": "unresolved"}, format="json"
+            url, {"assignedTo": "user@doesnotexist.com", "status": "unresolved"}, format="json"
```

Comment:
In principle, it's bad form to use domains you don't control, sometimes I even include changes to use example.com/example.org/friends (there are a set of reserved domains -- https://datatracker.ietf.org/doc/html/rfc6761), but I didn't for this PR.